<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(39, 39, 39);</string>
  </property>
  <widget class="QLineEdit" name="emailField">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>250</y>
     <width>190</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 13pt &quot;Futura&quot;;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameField">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>220</y>
     <width>190</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter Username&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>220</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 13pt &quot;Futura&quot;;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(100, 35, 35, 255));
font: 57 italic 13pt &quot;Futura&quot;;
background-color: rgb(24, 118, 63);</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(100, 35, 35, 255));
font: 57 italic 13pt &quot;Futura&quot;;
background-color: rgb(24, 118, 63);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordField">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>280</y>
     <width>190</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;enter Password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 13pt &quot;Futura&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passRepeatField">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>310</y>
     <width>190</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="passRepeatLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 13pt &quot;Futura&quot;;</string>
   </property>
   <property name="text">
    <string>Repeat Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>20</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/Desktop/Снимок экрана 2020-10-25 в 14.38.00.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
